suites/nautilus/rgw/tier-2_rgw_ganesha.yaml

```diff
@@ -0,0 +1,269 @@
+      desc: test cluster setup using ceph-ansible
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
+      name: check-ceph-health
+      module: exec.py
+      config:
+        cmd: ceph -s
+        sudo: true
+      desc: Check for ceph health debug info
+  - test:
+      name: NFS V4 test1
+      desc: Test and Verify NFS I/O Create operation
+      abort-on-fail: true
+      module: sanity_rgw_ganesha.py
+      polarion-id: CEPH-83574752
+      config:
+        script-name: test_on_nfs_io.py
+        config-file-name: test_on_nfs_io_create.yaml
+        nfs-version: 4  # To be added in rgw_user.yaml
+        test-config:
+          basedir_count: 4
+          subdir_count: 10
+          file_count: 10
+          objects_size_range:
+            min: 5
+            max: 10
+          io_op_config:
+            create: true
+  - test:
+      name: NFS V4 test2
+      desc: Test and Verify NFS I/O Delete operation
+      abort-on-fail: true
+      module: sanity_rgw_ganesha.py
+      polarion-id: CEPH-83574753
+      config:
+        script-name: test_on_nfs_io.py
+        config-file-name: test_on_nfs_io_delete.yaml
+        nfs-version: 4  # To be added in rgw_user.yaml
+        test-config:
+          basedir_count: 5
+          subdir_count: 10
+          file_count: 10
+          objects_size_range:
+            min: 5
+            max: 10
+          io_op_config:
+            create: true
+            delete: true
+  - test:
+      name: NFS V4 test3
+      desc: Test and Verify NFS I/O Move operation
+      abort-on-fail: true
+      module: sanity_rgw_ganesha.py
+      polarion-id: CEPH-83574754
+      config:
+        script-name: test_on_nfs_io.py
+        config-file-name: test_on_nfs_io_move.yaml
+        nfs-version: 4  # To be added in rgw_user.yaml
+        test-config:
+          basedir_count: 8
+          subdir_count: 3
+          file_count: 50
+          objects_size_range:
+            min: 5
+            max: 10
+          io_op_config:
+            create: true
+            move: true
+  - test:
+      name: NFS V3 test1
+      desc: Test and Verify NFS I/O Create operation
+      abort-on-fail: true
+      module: sanity_rgw_ganesha.py
+      polarion-id: CEPH-83574755
+      config:
+        script-name: test_on_nfs_io.py
+        config-file-name: test_on_nfs_io_create.yaml
+        nfs-version: 3  # To be added in rgw_user.yaml
+        test-config:
+          basedir_count: 4
+          subdir_count: 10
+          file_count: 10
+          objects_size_range:
+            min: 5
+            max: 10
+          io_op_config:
+            create: true
+  - test:
+      name: NFS V3 test2
+      desc: Test and Verify NFS I/O Delete operation
+      abort-on-fail: true
+      module: sanity_rgw_ganesha.py
+      polarion-id: CEPH-83574756
+      config:
+        script-name: test_on_nfs_io.py
+        config-file-name: test_on_nfs_io_delete.yaml
+        nfs-version: 3  # To be added in rgw_user.yaml
```

Comment:
Since NFSv3 is not a high priority, we can have a separate YAML may be or move it to tier-3 for better comparison of test results between the 2 protocol versions and for reducing the runtime in the pipeline as well.

Reply:
updated the changes and created a new yaml.